ui/lib/ve-token.ts

```diff
@@ -4,45 +4,46 @@ import { veNationToken } from '../lib/config'
 import { useBalance, useContractRead, useContractWrite } from './use-wagmi'
 
 const contractParams = {
-  addressOrName: veNationToken,
-  contractInterface: VotingEscrow.abi,
+  address: veNationToken,
+  abi: VotingEscrow.abi,
 }
 
 export function useVeNationBalance(address: any) {
   return useBalance({
-    addressOrName: address,
+    address,
     token: veNationToken,
     watch: true,
-    enabled: address,
+    enabled: !!address,
```

Comment:
`!!address`
@en0c-026 What do two exclamation marks indicate?

Reply:
convert the address string in a boolean value